src/components/notifier/index.tsx

```diff
@@ -16,6 +17,13 @@ export class Notifier {
 			this.notice = undefined
 		}
 	}
+	@Watch("notice")
+	onUpdatedNotice(newValue: string | Notice) {
+		if (newValue != undefined)
+			this.timer = window.setTimeout(() => { this.notice = undefined }, 5000)
+		else
+			window.clearTimeout(this.timer)
```

Comment:
`setTimeout` executes only once, then `clearTimeout` should not be needed, or?

Reply:
I worried about some use case where a timer from a previous notice might influence a new notice, but maybe that can't happen if I just reset the same timer.

Reply:
It's still possible to click the events away, and then the timer keeps running for a bit.

Reply:
That is actually the problem you are creating. If you want to avoid that then you should look if 'this.timer' is set before setting it again and clear it in that case (and also assign undefined when the timer fires).

Reply:
*notices

Reply:
So you indirectly clear the timer that way. That is acceptable but you should probably check if the timer exists before clearing it, or?

Reply:
When I try removing the clear, I can create this situation:
Get a notice. Click it away. Make a new change to get another notice. The notice disappears after remaining timer.

Reply:
I couldn't see your messages before reloading the page. I'm trying you method now.

Reply:
Removing the clear is not the issue, we concluded that it is needed. The issue is that you don't clear an ongoing timer when a new notification arrives and that you do try to clear timers already timed out.

The underlying problem is that you don't handle multiple notifications simultaneously. A new notification will remove the old one.

Reply:
Yes, I think we thought about multiple notifications earlier and went with the "this is okay for now" approach earlier.